src/libmongoc/doc/howto/index.rst

```diff
@@ -0,0 +1,15 @@
+##################
+Cookbook & How-Tos
```

Comment:
The use of the term "Cookbook" is somewhat confusing. Suggest removing it in favor of just "How-Tos".

Reply:
This one I must have subconsciously borrowed from TDS "recipes" analogy, but I agree that it is a strange term. Will change.